packages/graphql-yoga/src/index.ts

```diff
@@ -207,4 +185,9 @@ export {
   useTiming,
   EnvelopError as GraphQLServerError,
 } from '@envelop/core'
-export { GraphQLUpload } from 'graphql-upload'
+
+export const GraphQLUpload = new GraphQLScalarType({
```

Comment:
This doesn't matter. It is just a noop scalar that returns given value as is, it is `File` in our case.

Reply:
yeah good enough for now. We just want to export something to make it easier for users to use it.